packages/autocomplete-js/src/__tests__/autocomplete.test.ts

```diff
@@ -75,6 +75,7 @@ describe('autocomplete-js', () => {
               </label>
               <div
                 class="aa-LoadingIndicator"
+                hidden=""
```

Comment:
Since we don't manually use `requestAnimationFrame` now (we let Preact do the work), these `hidden` attributes now exist during first render.